### DISCUSSION TITLE
Is there a docker image for carbon?

### DISCUSSION THREAD

**Discussion by liorp** [dawnlabs/carbon#1378] [Q&A] [answered]
Hi,

Carbon is literally the best thing to happen to code sharing.

Is there a docker image available for local deployment?

Thanks!

**Comment by mfix22** [comment, marked as answer]
@liorp I know folks have deployed unofficial versions of Carbon to Docker hub in the past, but we don't have an official 1st-party version. You should be able to create your own Docker image fairly easily, though.
